hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/HoodieDeltaStreamer.java

```diff
@@ -276,7 +276,17 @@ public static class Config implements Serializable {
             + ". Allows transforming raw source Dataset to a target Dataset (conforming to target schema) before "
             + "writing. Default : Not set. E:g - org.apache.hudi.utilities.transform.SqlQueryBasedTransformer (which "
             + "allows a SQL query templated to be passed as a transformation function). "
-            + "Pass a comma-separated list of subclass names to chain the transformations.")
+            + "Pass a comma-separated list of subclass names to chain the transformations. Transformer can also include "
+            + "an identifier. E:g - tr1:org.apache.hudi.utilities.transform.SqlQueryBasedTransformer. Here the identifier tr1 "
+            + "can be used along with property key like `hoodie.deltastreamer.transformer.sql.tr1` to identify properties related "
+            + "to the transformer. So effective value for `hoodie.deltastreamer.transformer.sql` is determined by key "
+            + "`hoodie.deltastreamer.transformer.sql.tr1` for this transformer. This is useful when there are two or more "
+            + "transformers using the same config keys and expect different values for those keys. If identifier is used, it should "
+            + "be specified for all the transformers. Further the order in which transformer is applied is determined by the occurrence "
+            + "of transformer irrespective of the identifier used for the transformer. For example: In the configured value below "
+            + "tr2:org.apache.hudi.utilities.transform.SqlQueryBasedTransformer,tr1:org.apache.hudi.utilities.transform.SqlQueryBasedTransformer "
+            + ", tr2 is applied before tr1 based on order of occurrence."
+    )
```

Comment:
can we call out that this identifier format is not strictly required unless users have a requirement to have multiple transformers of the same type.

Reply:
Addressed